### DISCUSSION TITLE
Why the instruction to install openvpn?

### DISCUSSION THREAD

**Discussion by madalinignisca** [EnterpriseDB/tpa#189] [General]
I'm very confused on what is the usage of the `openvpn` package for using TPA? What is the relation between them? Is it a hard dependency?

**Comment by haroon-github** [comment]
Can you add a little more detail on where you see openvpn as a dependency?

**Comment by gonzalemario** [comment]
@madalinignisca openvpn is not a hard dependency. Did you find some bit that says otherwise?

**Reply by madalinignisca** [reply]
I know it is not, but the documentation might confuse people:

https://www.enterprisedb.com/docs/tpa/latest/INSTALL-repo/

<img width="700" alt="Screenshot 2024-12-13 at 22 51 44" src="https://github.com/user-attachments/assets/56d46886-ac96-4cbc-8e70-898012844318" />

**Reply by gonzalemario** [reply]
I see what you mean, thanks!
